garnetred/loc-collective
# Feat/page cleanup

#### What's this PR do?
This PR creates a new footer that links to my business website. I also made a few other minor changes to text and added some tests. 
Additionally, I took this time to clean up some components. There were WebContentContainer and WebContent components to make a reusable text component. However, the pages that used both components (About and Style) had different use cases. In hindsight the component was made reusable prematurely. Instead I deleted those components and created two separate About and Style components, along with css files and tests. 

#### Where should the reviewer start?
I would start with looking at the footer and running the tests (there are some failing tests due to the loading spinner). Next I would check for text updates. I'm hoping to update the responsive design in another PR. You can also check the about and style pages, which should look the same as before. 
 
#### How should this be manually tested?
You can clone the repo, run the application via the command `npm run start`, and check that the footer appears on all pages. You can also check the about and style pages, which should look the same as before. 

#### Any background context you want to provide?
Just making some changes now that the site is publicly available. 

#### What are the relevant tickets?

#### Screenshots (if appropriate)
Demo

https://user-images.githubusercontent.com/59572865/227056126-37bbf0fd-b03f-4036-823d-d1327000debb.mov

#### Questions
